docs/book/helpers/html-attributes.md

```diff
@@ -0,0 +1,161 @@
+# HtmlAttributes
+
+The `HtmlAttributes` helper can be used to make code related to HTML attributes
+easier to write and read. The helper initializes and returns
+`Laminas\View\HtmlAttributesSet` object instances, which can then be manipulated
+and converted to strings.
+
+## Basic Usage
+
+```php
+<?php
+$attributes = $this->htmlAttributes(['class' => 'input-group']);
+if ($form->hasValidated()) {
+    $attributes->add('class', 'has-validation');
+}
+?>
+
+<div<?= $attributes ?>></div>
+```
+
+Output:
+
+```html
+<div class="input-group&#x20;has-validation"></div>
+```
+
+## Getting an `HtmlAttributesSet` object instance
+
+To get an empty `HtmlAttributesSet` object instance, call the helper without any
+parameters.
```

Comment:
Documentation should be written using the [one sentence per-line method](https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line).

```suggestion
To get an empty `HtmlAttributesSet` object instance, call the helper without any parameters.
```